TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1294,26 +1391,39 @@ for i = 1, 40 do
 end
 
 spec:RegisterHook( "COMBAT_LOG_EVENT_UNFILTERED", function( _, subtype, _, sourceGUID, sourceName, _, _, destGUID, destName, destFlags, _, spellID, spellName )
```

Comment:
```Lua
local buffLeaderPackCurrent = 0
```

```Lua
spec:RegisterSpecFunction( "buff_leader_pack_current", function() return buffLeaderPackCurrent end )

-- in reset_precast:
buff_leader_pack_current = nil
```

Reply:
(Or `pack_leader_buff_current`  I didn't intentionally change the words around.)